build.sbt

```diff
@@ -197,13 +197,14 @@ lazy val zincCompile = (project in file("zinc-compile"))
   )
   .configure(addSbtUtilTracking)
 
-// Persists the incremental data structures using SBinary
+// Persists the incremental data structures using Protobuf
 lazy val zincPersist = (project in internalPath / "zinc-persist")
   .dependsOn(zincCore, zincCore % "test->test")
   .configure(addBaseSettingsAndTestDeps)
   .settings(
     name := "zinc Persist",
-    libraryDependencies += sbinary
+    compileOrder := sbt.CompileOrder.Mixed,
```

Comment:
Because Java sources depend on Scala sources, and viceversa. It's pretty common in Zinc modules.